roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -23,7 +23,7 @@
   when: openshift.common.is_containerized | bool and not openshift.common.is_master_system_container | bool
 
 # workaround for missing systemd unit files
-- name: Create the systemd unit files
+- name: "Create the {{ openshift.common.service_type }} systemd unit file"
```

Comment:
Vars in task names have been around for quite a long time. I can confirm because the original bug report for it was against a module I wrote like 3 years ago which turned into a larger "oh snap, we need to evaluate those statements everywhere" bug.